From: Thomas Hindoe Paaboel Andersen <phomes@gmail.com>
Date: Tue, 17 Dec 2013 20:15:45 +0100
Subject: journal-file.c: remove redundant assignment of variable

we also do 'last_index = (uint64_t) -1;' at the end of the while
loop so there is no reason to also do it here.

(cherry picked from commit 0d647f7f73fa17074a1df81e2bdd00877554e436)
---
 src/journal/journal-file.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/journal/journal-file.c b/src/journal/journal-file.c
index 71ef092e..5ef6a2a7 100644
--- a/src/journal/journal-file.c
+++ b/src/journal/journal-file.c
@@ -1628,8 +1628,6 @@ static int generic_array_bisect(
                                         else
                                                 left = y + 1;
                                 }
-
-                                last_index = (uint64_t) -1;
                         }
 
                         for (;;) {
